client/db/bolt/db.go

```diff
@@ -52,7 +52,8 @@ var (
 	appBucket              = []byte("appBucket")
 	accountsBucket         = []byte("accounts")
 	disabledAccountsBucket = []byte("disabledAccounts")
-	ordersBucket           = []byte("orders")
+	ordersBucket           = []byte("activeOrders")
```

Comment:
What to you think about renaming the `const` to `activeOrdersBucket`?

Reply:
Works for me!